spec/pom.xml

```diff
@@ -37,12 +37,12 @@
     </licenses>
     <issueManagement>
         <system>github</system>
-        <url>https://github.com/eclipse-ee4j/ejb-api/issues</url>
+        <url>https://github.com/jakartaee/enterprise-beans/issues</url>
     </issueManagement>
     <scm>
-        <connection>scm:git:ssh://git@github.com/eclipse-ee4j/ejb-api.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/eclipse-ee4j/ejb-api.git</developerConnection>
-        <url>https://github.com/eclipse-ee4j/ejb-api/tree/${project.scm.tag}</url>
+        <connection>scm:git:ssh://git@github.com/jakartaee/enterprise-beans.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/jakartaee/enterprise-beans.git</developerConnection>
+        <url>https://github.com/jakartaee/enterprise-beans/tree/${project.scm.tag}</url>
```

Comment:
```suggestion
        <url>https://github.com/jakartaee/enterprise-beans</url>
```

Reply:
I think having the project home page should suffice for now, and so I removed tag element from the url.